### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by bsonge**
Merging base develop with master branch.  This should be the "v1" of the project that we showed off in class.  I'd like to go ahead and update master before we move on.

### CODE REVIEW

**Comment by Quinn-Donnelly** [comment]
What is the status on the test cases for the components we have built so far?

File: app/components/BigBtn1/index.js
Comment (Quinn-Donnelly) [inline app/components/BigBtn1/index.js:48]: Unsure how this component is being used however I think a few more props are probably required.

File: app/components/BigBtn1/tests/index.test.js
Comment (Quinn-Donnelly) [inline app/components/BigBtn1/tests/index.test.js]: I will also post once by the tests I think we should write some before we go onto to other work or merge to master.

File: app/components/Button1/index.js
Comment (Quinn-Donnelly) [inline app/components/Button1/index.js]: If we aren't using this commented code we should purge it for clean code.

File: app/components/Button1/messages.js
Comment (Quinn-Donnelly) [inline app/components/Button1/messages.js:11]: Careful with our messages being informal (flashback to phase one report)

File: app/components/DbEntry/index.js
Comment (Quinn-Donnelly) [inline app/components/DbEntry/index.js:18]: Not sure which naming convention we are using for function names but we should decide on one. Also the fill function seems to be useless in it's current state.

File: app/components/DbEntry/index.js
Comment (Quinn-Donnelly) [inline app/components/DbEntry/index.js:46]: Same as above vestigial code should either be removed or commented to explain what it is there for.

File: app/containers/HomePage/index.js
Comment (Quinn-Donnelly) [inline app/containers/HomePage/index.js]: If we are linking in this manner then it is especially important that we write test cases to verify these links so we don't forget about them when we make changes to their respective pages.

File: app/containers/NavBar/index.js
Comment (Quinn-Donnelly) [inline app/containers/NavBar/index.js:38]: These will be important to have tests for to check unverified links.

File: app/containers/Registration/index.js
Comment (Quinn-Donnelly) [inline app/containers/Registration/index.js]: For text the messages file should be used so that we can use the Language Provider.

File: test_html/button1.html
Comment (Quinn-Donnelly) [inline test_html/button1.html:1]: Not sure what this file or directory are being used for.

**Review by Quinn-Donnelly — CHANGES_REQUESTED** [review]
I have reviewed the changes and comments on the files where I found thinks that might need to be changed. If it was a reoccurring change that needed to be made I only commented once about it but it could appear in multiple locations.

**Comment by bsonge** [comment]
I'll review the changes and get fixes going.  We started test cases yesterday and realized that if we do only test cases we might not get enough time to actually implement phase II, so I think the consensus is to start on phase II changes and write tests as we go so that we get used to it, then co back if we have time.

**Pull request closed without merging**
